rpc/grpc/client/client.go

```diff
@@ -50,13 +52,18 @@ func defaultDialerFunc(ctx context.Context, addr string) (net.Conn, error) {
 }
 
 type client struct {
-	conn grpc.ClientConn
+	conn *ggrpc.ClientConn
```

Comment:
Not sure why this `grpc.ClientConn` from the gogoproto fork was pulled in - could've been done automatically by my IDE.

Reply:
Do you know what the diff is, this might be the cause of Lauren's problems maybe? :) anyways, PR looks good to me